publisher/monitor/balance/monitor-balance.py

```diff
@@ -15,7 +15,11 @@
 
 
 async def main(publishers=None, threshold_wei=None):
-    # Set admin private key to None because we aren't using the client for protected invokes
+    slack_url = "https://slack.com/api/chat.postMessage"
```

Comment:
huge value add to have in slack. Big keep for socializing alert response

Reply:
👍  Combined with the deadman switch on betteruptime we also get even better visibility. betteruptime only -> network/completion issue; betteruptime + slack -> issue on our end/actual error